teps/0084-endtoend-provenance-collection.md

```diff
@@ -188,13 +129,254 @@ The `in-toto` format created by chains wraps the [SLSA Provenance v0.2](https://
   * **results**: An array of objects representing the results of a `taskrun`. Each object contains the attribute `name` and `value`. Their values are of type string. `taskrun` results are important in the context of a `pipelinerun` because `taskrun` results may be used by other tasks, thus impacting the output of the `pipelinerun`.
 * **predicate.materials**: For `taskruns`, Chains populates this object based on specially named task parameters or results, i.e. `CHAINS-GIT_COMMIT` and `CHAINS-GIT_URL`. Similarly, Chains must populate this object with the corresponding pipeline parameters or results.
 
+<details>
+    <summary>Click to view an example pipelinerun attestation</summary>
```

Comment:
Do you mind to add pipeline and pipelinerun definition here or the links to them? That would be really helpful for the readers of the TEP to map the fields in the attestation.
Thanks

Reply:
Yes, good idea. Added.